pkgs/tools/graphics/argyllcms/default.nix

```
@@ -25,109 +101,62 @@ stdenv.mkDerivation rec {
     }
   );
 
-  postPatch = lib.optionalString (stdenv.hostPlatform != stdenv.buildPlatform) ''
-    substituteInPlace Jambase \
-      --replace "-m64" ""
-  '';
-
-  preConfigure = let
-    # The contents of this file comes from the Jamtop file from the
-    # root of the ArgyllCMS distribution, rewritten to pick up Nixpkgs
-    # library paths. When ArgyllCMS is updated, make sure that changes
-    # in that file is reflected here.
-    jamTop = writeText "argyllcms_jamtop" ''
-      DESTDIR = "/" ;
-      REFSUBDIR = "share/argyllcms" ;
-
-      # Keep this DESTDIR anchored to Jamtop. PREFIX is used literally
-      ANCHORED_PATH_VARS = DESTDIR ;
-
-      # Tell standalone libraries that they are part of Argyll:
-      DEFINES += ARGYLLCMS ;
-
-      # enable serial instruments & support
-      USE_SERIAL = true ;
-
-      # enable fast serial instruments & support
-      USE_FAST_SERIAL = true ;                # (Implicit in USE_SERIAL too)
-
-      # enable USB instruments & support
-      USE_USB = true ;
-
-      # enable dummy Demo Instrument (only if code is available)
-      USE_DEMOINST = true ;
+  postPatch = ''
+      substituteInPlace Makefile --replace "-j 3" "-j $NIX_BUILD_CORES"
+      substituteInPlace Jamtop --replace "EXIT Unable" "ECHO Unable"
```

Use of `--replace-fail` (or `--replace-warn` or `--replace-quiet) is preferred over `--replace`:
```suggestion
      substituteInPlace Makefile --replace-fail "-j 3" "-j $NIX_BUILD_CORES"
      substituteInPlace Jamtop --replace-fail "EXIT Unable" "ECHO Unable"
```